provider/marathon/marathon.go

```diff
@@ -369,6 +449,62 @@ func (p *Provider) hasCircuitBreakerLabels(application marathon.Application) boo
 	return ok
 }
 
+// Map of services properties
+// we can get it with label[serviceName][propertyName] and we got the propertyValue
+type servicePropertyValues map[string]string
+
+type labelServiceProperties map[string]servicePropertyValues
+
+// Check if for the given application, we find labels that are defining services
+func (p *Provider) hasServices(application marathon.Application) bool {
```

Comment:
One usecase that we have is a different front-ends (completely) depending on if the container has multiple services exposed or not.  For multi-service containers, we need to add some extra rules for adding header values, etc...